crates/core/tedge_actors/src/errors.rs

```diff
@@ -32,6 +32,9 @@ pub enum RuntimeError {
 
     #[error(transparent)]
     LinkError(#[from] LinkError),
+
+    #[error(transparent)]
+    HealthMonitorError(#[from] anyhow::Error),
```

Comment:
Why are we adding an actor specific error to the core actor crate? If a health-check related error really needs to be a first class citizen in the actor crate (I doubt that's the case), then better to define that as a concrete type rather than as a wrapper over `anyhow:Error`.